README.md

```diff
@@ -1,6 +1,6 @@
 # Diagnostics Working Group
 
-[![Join us on nodeslackers.com (channel #diagnostics-wg)](https://img.shields.io/badge/Join%20us%20on-nodeslackers.com-blue)](https://node-js.slack.com/archives/C016U31J15M)
+[![Join us on OpenJS Foundation slack (channel #nodejs-diagnostics-wg)](https://img.shields.io/badge/Join%20us%20on-nodeslackers.com-blue)](https://openjs-foundation.slack.com/archives/C025TDEBHN1)
```

Comment:
The image should probably be updated as well.

![image](https://img.shields.io/badge/Join%20us%20on-nodeslackers.com-blue)